CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 1.2.0 (June 28, 2023)
```

Comment:
There was talks around consolidating the `RC` notes with `1.2.0` so that it's clearer what `1.2.0` offers. I would just check with PM @david-yu @blake

Reply:
I think that would make things clearer.